src/flags/blocks.rs

```diff
@@ -181,6 +183,7 @@ impl TryFrom<&str> for Block {
             "date" => Ok(Self::Date),
             "name" => Ok(Self::Name),
             "inode" => Ok(Self::INode),
+            "inode_count" => Ok(Self::INodeCount),
```

Comment:
`inode-count`.

Reply:
When just long is passed, don't show it. I think we'll keep the current default long blocks as they are for now.

Reply:
Right now it doesn't show if only long is passed, but only shows the links if blocks is passed.